libraries/AP_Scripting/examples/quadruped.lua

```diff
@@ -37,23 +37,26 @@ local body_pos_y = 0    -- body position in the Y axis (i.e. right, left).  shou
 local body_pos_z = 0    -- body position in the Z axis (i.e. up, down).  should be -40mm to +40mm
 
 -- starting positions of the legs
-local endpoints1 = {math.cos(math.rad(45))*(COXA_LEN + FEMUR_LEN), math.sin(math.rad(45))*(COXA_LEN + FEMUR_LEN), TIBIA_LEN}
-local endpoints2 = {math.cos(math.rad(45))*(COXA_LEN + FEMUR_LEN), math.sin(math.rad(-45))*(COXA_LEN + FEMUR_LEN), TIBIA_LEN}
-local endpoints3 = {-math.cos(math.rad(45))*(COXA_LEN + FEMUR_LEN), math.sin(math.rad(-45))*(COXA_LEN + FEMUR_LEN), TIBIA_LEN}
-local endpoints4 = {-math.cos(math.rad(45))*(COXA_LEN + FEMUR_LEN), math.sin(math.rad(45))*(COXA_LEN + FEMUR_LEN), TIBIA_LEN}
+local endpoint_LB = {math.cos(math.rad(45))*(COXA_LEN + FEMUR_LEN), math.sin(math.rad(45))*(COXA_LEN + FEMUR_LEN), TIBIA_LEN}
+local endpoint_LF = {math.cos(math.rad(45))*(COXA_LEN + FEMUR_LEN), math.sin(math.rad(-45))*(COXA_LEN + FEMUR_LEN), TIBIA_LEN}
+local endpoint_RF = {-math.cos(math.rad(45))*(COXA_LEN + FEMUR_LEN), math.sin(math.rad(-45))*(COXA_LEN + FEMUR_LEN), TIBIA_LEN}
+local endpoint_RB = {-math.cos(math.rad(45))*(COXA_LEN + FEMUR_LEN), math.sin(math.rad(45))*(COXA_LEN + FEMUR_LEN), TIBIA_LEN}
 
 -- control input enum
 local control_input_roll = 1
 local control_input_pitch = 2
 local control_input_throttle = 3
 local control_input_yaw = 4
+local control_input_height = 8
+
+local xy_travel_max = 80     -- x and y axis travel max (used to convert control input) in mm
+local yaw_lenght_max = 10    -- yaw travel maximum (used to convert control input)
```

Comment:
small typo, should be "yaw_length_max" (t and h are reversed)

Reply:
is it yaw_length_max or yaw_travel_max?

Reply:
fixed to yaw_travel_max , thanks